great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -525,27 +527,12 @@ def _get_bootstrap_estimate(
             random_seed=random_seed,
         )
 
+    @staticmethod
     def _get_deterministic_estimate(
-        self,
         metric_values: np.ndarray,
-        domain: Domain,
-        *,
-        variables: Optional[ParameterContainer] = None,
-        parameters: Optional[Dict[str, ParameterContainer]] = None,
         **kwargs,
     ) -> Tuple[Number, Number]:
-        # Obtain false_positive_rate from "rule state" (i.e., variables and parameters); from instance variable otherwise.
-        false_positive_rate: np.float64 = get_parameter_value_and_validate_return_type(
-            domain=domain,
-            parameter_reference=kwargs.get("false_positive_rate", 5.0e-2),
-            expected_return_type=(float, np.float64),
-            variables=variables,
-            parameters=parameters,
-        )
-        if not (0.0 <= false_positive_rate <= 1.0):
-            raise ge_exceptions.ProfilerExecutionError(
-                message=f"The confidence level for {self.__class__.__name__} is outside of [0.0, 1.0] closed interval."
-            )
+        false_positive_rate: np.float64 = kwargs.get("false_positive_rate", 5.0e-2)
```

Comment:
And we're confident that if "false_positive_rate" is a key in kwargs, it's value will be a `np.float64` right? Just confirming since I want to ensure this value is the desired type.

Reply:
@cdkini As a type check it works for all floating point types.